src/emulator/types.ts

```diff
@@ -153,37 +149,13 @@ export class EmulatorLog {
   }
 
   /**
-   * As discovered in #1486, some very large log messages (>8192B) were not being properly passed
-   * between the emulator runtime and the emulator itself. We were falsely making the assumption
-   * that we could pass messages of any length over stdout and the stream reader would always get
-   * a whole message in a single "data" callback.
-   *
-   * Now we chunk the messages into 8000B pieces and then add a signal (CHUNK_DIVIDER) to the
-   * end of partial messages that instructs the receiver to wait for the whole message to
-   * appear.
-   *
    * We use a global boolean to know if all of our messages have been flushed, and the functions
    * emulator can wait on this variable to flip before exiting. This ensures that we never
-   * miss a log message that has been queued but has not yet flushed from stdout.
-   *
-   * Note: it would be better to use ipc via process.send() since that is faster and has
-   * extremely large message limits but in some experiments the IPC channel seemed to get
-   * full and not flush, so stdout remains.
+   * miss a log message that has been queued but has not yet flushed.
    */
   log(): void {
     const msg = `${this.toString()}\n`;
-    const chunks = this.chunkString(msg);
```

Comment:
We can remove this whole chunking system because node IPC has no message size limit (unlike stdout)